src/doc/rustc/src/platform-support/fuchsia.md

```diff
@@ -641,8 +641,60 @@ available on the [Fuchsia devsite].
 
 ### Running the compiler test suite
 
-Running the Rust test suite on Fuchsia is [not currently supported], but work is
-underway to enable it.
+Pre-requisites for running the Rust test suite on Fuchsia are:
+1. Installation of `rustc` via "[Targeting Fuchsia with a compiler built from source](#targeting-fuchsia-with-a-compiler-built-from-source)", including `config-env.sh` and `config.toml`.
+1. Checkout of Rust source.
+1. Download of the Fuchsia SDK. Minimum supported SDK version is [4xjxrGUrDbQ6_zJwj6cDN1IbWsWV5aCQXC_zO_Hu0XkC](https://chrome-infra-packages.appspot.com/p/fuchsia/sdk/core/linux-amd64/+/4xjxrGUrDbQ6_zJwj6cDN1IbWsWV5aCQXC_zO_Hu0XkC)
```

Comment:
There is a human-readable version tag on that page: [9.20220726.1.1](https://chrome-infra-packages.appspot.com/p/fuchsia/sdk/core/linux-amd64/+/version:9.20220726.1.1). Let's use that since it can be meaningfully compared to other versions.